packages/babel-plugin-proposal-class-properties/test/fixtures/private-loose/destructuring-array-pattern-1/output.js

```diff
@@ -0,0 +1,14 @@
+var Foo = function Foo(props) {
+  "use strict";
+
+  babelHelpers.classCallCheck(this, Foo);
+  Object.defineProperty(this, _client, {
+    writable: true,
+    value: void 0
+  });
+  babelHelpers.classPrivateFieldLooseBase(this, _client)[_client] = 1;
+  ;
```

Comment:
Where does this extra semicolon come from?

Reply:
oh, it's due to an extra comma in the `input.js`

Reply:
removed extra comma